### ISSUE TITLE
docker: Error response from daemon: OCI runtime create failed: flag provided but not defined: -console-socket: unknown.

### ISSUE THREAD

**Issue by ccbt87** [HabanaAI/Model-References#5]
When we try to run the TensorFlow and PyTorch containers, we got the error message `flag provided but not defined: -console-socket: unknown`.

```
root@localhost:~# docker run -it --runtime=habana -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_vader_single_copy_mechanism=none --cap-add=sys_nice -v /sys/kernel/debug:/sys/kernel/debug --net=host -v $HOME/hlogs:/root/hlogs -v $HOME/tmp:/root/tmp -v /software/data/tf/data:/software/data/tf/data -v /software/data/bert_checkpoints:/software/data/bert_checkpoints vault.habana.ai/gaudi-docker/0.14.0/ubuntu18.04/habanalabs/tensorflow-installer-tf-cpu-2.4.1:0.14.0-420
docker: Error response from daemon: OCI runtime create failed: flag provided but not defined: -console-socket: unknown.
ERRO[0000] error waiting for container: context canceled
```

Please advise

**Comment by greg-serochi** [comment]
Hi, thanks for post this question.   It is under review internally.

**Comment by mspandit** [comment]
Resolved by upgrading to 0.14.
